**(tests) - optaweb-vehicle-routing job** `#9` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/9/display/redirect

**Test results:**
- PASSED: 247
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/9/testReport/org.optaweb.vehiclerouting.plugin.routing/GraphHopperIntegrationTest/graphhopper_should_import_and_load_osm_file_successfully(Path)/">org.optaweb.vehiclerouting.plugin.routing.GraphHopperIntegrationTest.graphhopper_should_import_and_load_osm_file_successfully(Path)</a></summary>
java.lang.AssertionError: <br/><br/>Expecting code not to raise a throwable but caught<br/>  "java.lang.RuntimeException: Graph after reading OSM must not be empty. Read 1 items and 0 locations<br/>	at com.graphhopper.reader.osm.OSMReader.writeOsm2Graph(OSMReader.java:299)<br/>	at com.graphhopper.reader.osm.OSMReader.readGraph(OSMReader.java:146)<br/>	at com.graphhopper.GraphHopper.importData(GraphHopper.java:682)<br/>	at com.graphhopper.GraphHopper.process(GraphHopper.java:653)<br/>	at com.graphhopper.GraphHopper.importOrLoad(GraphHopper.java:631)<br/>	at org.assertj.core.api.ThrowableAssert.catchThrowable(ThrowableAssert.java:63)<br/>	at org.assertj.core.api.AssertionsForClassTypes.catchThrowable(AssertionsForClassTypes.java:878)<br/>	at org.assertj.core.api.AssertionsForClassTypes.assertThatCode(AssertionsForClassTypes.java:849)<br/>	at org.assertj.core.api.Assertions.assertThatCode(Assertions.java:1258)<br/>	at org.optaweb.vehiclerouting.plugin.routing.GraphHopperIntegrationTest.graphhopper_should_import_and_load_osm_file_successfully(GraphHopperIntegrationTest.java:41)<br/>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>	at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/>	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)<br/>	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)<br/>	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)<br/>	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)<br/>	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)<br/>	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)<br/>	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)<br/>	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)<br/>"<br/>	at org.optaweb.vehiclerouting.plugin.routing.GraphHopperIntegrationTest.graphhopper_should_import_and_load_osm_file_successfully(GraphHopperIntegrationTest.java:41)<br/>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>	at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/>	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)<br/>	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)<br/>	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)<br/>	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)<br/>	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)<br/>	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)<br/>	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)<br/>	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)<br/>	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)<br/>	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)<br/>	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)<br/>	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)<br/>	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)<br/>	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)<br/>	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)<br/>	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)<br/>	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)<br/>	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)<br/>	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)<br/>
</details>

See console log:
<details>
<summary><b>Console Logs</b></summary>
[2023-04-28T07:49:18.829Z] [INFO] --- maven-dependency-plugin:3.1.2:tree (default-cli) @ optaweb-vehicle-routing-docs ---<br/>[2023-04-28T07:49:18.829Z] [INFO] org.optaweb.vehiclerouting:optaweb-vehicle-routing-docs:pom:8.13.1-SNAPSHOT<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --< org.optaweb.vehiclerouting:optaweb-vehicle-routing-distribution >---<br/>[2023-04-28T07:49:18.829Z] [INFO] Building OptaWeb Vehicle Routing Distribution 8.13.1-SNAPSHOT      [6/6]<br/>[2023-04-28T07:49:18.829Z] [INFO] --------------------------------[ pom ]---------------------------------<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --- formatter-maven-plugin:2.11.0:validate (default) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:18.829Z] [INFO] Using 'UTF-8' encoding to format source files.<br/>[2023-04-28T07:49:18.829Z] [INFO] Number of files to be formatted: 0<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-no-logback-test-in-main) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (no-managed-deps) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:18.829Z] [INFO] <br/>[2023-04-28T07:49:18.829Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (ban-forbidden-dependencies) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (no-duplicate-declared-dependencies) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java-version) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-maven-version) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (jacoco-prepare-agent) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] jacoco.agent.argLine set to -javaagent:/home/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/home/jenkins/workspace/KIE/kogito/1.13.x/pullrequest/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/bc/kiegroup_optaweb-vehicle-routing/optaweb-vehicle-routing/target/jacoco.exec,append=true,excludes=**/swingui/**/*<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- impsort-maven-plugin:1.8.0:check (default) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] Processed 0 files in 00:00.001 (Already Sorted: 0, Needed Sorting: 0)<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- maven-source-plugin:3.1.0:jar-no-fork (attach-sources) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] <br/>[2023-04-28T07:49:19.087Z] [INFO] --- maven-assembly-plugin:3.1.1:single (default) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:19.087Z] [INFO] Reading assembly descriptor: src/main/assembly/assembly-optaweb-vehicle-routing.xml<br/>[2023-04-28T07:49:20.970Z] [INFO] Building zip: /home/jenkins/workspace/KIE/kogito/1.13.x/pullrequest/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/bc/kiegroup_optaweb-vehicle-routing/optaweb-vehicle-routing/optaweb-vehicle-routing-distribution/target/optaweb-vehicle-routing-distribution-8.13.1-SNAPSHOT.zip<br/>[2023-04-28T07:49:26.208Z] [INFO] <br/>[2023-04-28T07:49:26.208Z] [INFO] --- maven-install-plugin:2.5.2:install (default-install) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:26.208Z] [INFO] Installing /home/jenkins/workspace/KIE/kogito/1.13.x/pullrequest/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/bc/kiegroup_optaweb-vehicle-routing/optaweb-vehicle-routing/optaweb-vehicle-routing-distribution/pom.xml to /home/jenkins/.m2/repository/org/optaweb/vehiclerouting/optaweb-vehicle-routing-distribution/8.13.1-SNAPSHOT/optaweb-vehicle-routing-distribution-8.13.1-SNAPSHOT.pom<br/>[2023-04-28T07:49:26.208Z] [INFO] Installing /home/jenkins/workspace/KIE/kogito/1.13.x/pullrequest/optaweb-vehicle-routing.tests.optaweb-vehicle-routing/bc/kiegroup_optaweb-vehicle-routing/optaweb-vehicle-routing/optaweb-vehicle-routing-distribution/target/optaweb-vehicle-routing-distribution-8.13.1-SNAPSHOT.zip to /home/jenkins/.m2/repository/org/optaweb/vehiclerouting/optaweb-vehicle-routing-distribution/8.13.1-SNAPSHOT/optaweb-vehicle-routing-distribution-8.13.1-SNAPSHOT.zip<br/>[2023-04-28T07:49:26.208Z] [INFO] <br/>[2023-04-28T07:49:26.208Z] [INFO] --- maven-dependency-plugin:3.1.2:tree (default-cli) @ optaweb-vehicle-routing-distribution ---<br/>[2023-04-28T07:49:26.208Z] [INFO] org.optaweb.vehiclerouting:optaweb-vehicle-routing-distribution:pom:8.13.1-SNAPSHOT<br/>[2023-04-28T07:49:26.208Z] [INFO] +- org.optaweb.vehiclerouting:optaweb-vehicle-routing-docs:zip:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO] \- org.optaweb.vehiclerouting:optaweb-vehicle-routing-standalone:zip:quarkus-app:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    +- org.optaweb.vehiclerouting:optaweb-vehicle-routing-backend:jar:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.13.4.2:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- com.fasterxml.jackson.core:jackson-annotations:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- org.yaml:snakeyaml:jar:1.33:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  \- com.fasterxml.jackson.core:jackson-core:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  +- io.quarkus:quarkus-hibernate-orm-panache:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- io.quarkus:quarkus-hibernate-orm:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- io.quarkus:quarkus-agroal:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.quarkus:quarkus-datasource:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  |  \- io.quarkus:quarkus-datasource-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.agroal:agroal-api:jar:1.16:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.agroal:agroal-narayana:jar:1.16:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  |  \- org.jboss:jboss-transaction-spi:jar:7.6.0.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  \- io.agroal:agroal-pool:jar:1.16:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- io.quarkus:quarkus-narayana-jta:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.quarkus:quarkus-transaction-annotations:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.smallrye:smallrye-context-propagation-jta:jar:1.2.2:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.smallrye.reactive:smallrye-reactive-converter-api:jar:2.7.0:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- io.smallrye.reactive:smallrye-reactive-converter-mutiny:jar:2.7.0:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- org.jboss.narayana.jta:narayana-jta:jar:5.13.1.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  |  \- org.jboss.spec.javax.resource:jboss-connector-api_1.7_spec:jar:1.0.0.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  \- org.jboss.narayana.jts:narayana-jts-integration:jar:5.13.1.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.hibernate:hibernate-core:jar:5.6.14.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- net.bytebuddy:byte-buddy:jar:1.12.18:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- antlr:antlr:jar:2.7.7:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- com.fasterxml:classmate:jar:1.5.1:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  \- org.hibernate.common:hibernate-commons-annotations:jar:5.1.2.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.hibernate:hibernate-graalvm:jar:5.6.14.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.glassfish.jaxb:jaxb-runtime:jar:2.3.3-b02:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- org.glassfish.jaxb:txw2:jar:2.3.3-b02:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  +- com.sun.istack:istack-commons-runtime:jar:3.0.10:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  |  \- com.sun.activation:jakarta.activation:jar:1.2.1:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- jakarta.persistence:jakarta.persistence-api:jar:2.2.3:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- jakarta.transaction:jakarta.transaction-api:jar:1.3.3:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.hibernate:quarkus-local-cache:jar:0.1.1:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- io.quarkus:quarkus-caffeine:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |     \- com.github.ben-manes.caffeine:caffeine:jar:2.9.3:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- io.quarkus:quarkus-hibernate-orm-panache-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- io.quarkus:quarkus-panache-hibernate-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  \- io.quarkus:quarkus-panache-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  +- io.quarkus:quarkus-jdbc-h2:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- com.h2database:h2:jar:2.1.214:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  \- org.locationtech.jts:jts-core:jar:1.17.0:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  +- io.quarkus:quarkus-jdbc-postgresql:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  \- org.postgresql:postgresql:jar:42.5.1:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  +- org.optaplanner:optaplanner-quarkus:jar:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- org.optaplanner:optaplanner-core:jar:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.drools:drools-alphanetwork-compiler:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  +- org.apache.commons:commons-math3:jar:3.6.1:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- io.micrometer:micrometer-core:jar:1.9.4:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |     +- org.hdrhistogram:HdrHistogram:jar:2.1.12:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |     \- org.latencyutils:LatencyUtils:jar:2.0.3:runtime<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- org.kie.kogito:kogito-legacy-api:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- org.kie.kogito:kogito-api:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |     \- org.kie:kie-api:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  +- org.kie.kogito:kogito-drools-model:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.208Z] [INFO]    |  |  |  \- org.drools:drools-model-compiler:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.kie:kie-internal:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.drools:drools-canonical-model:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- com.github.javaparser:javaparser-core:jar:3.23.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.drools:drools-mvel-parser:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     \- org.drools:drools-mvel-compiler:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- org.kie.kogito:drools-core:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- org.drools:drools-core:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  +- org.kie.soup:kie-soup-xstream:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  +- org.drools:drools-core-reflective:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  \- commons-codec:commons-codec:jar:1.15:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.thoughtworks.xstream:xstream:jar:1.4.19:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  \- io.github.x-stream:mxparser:jar:1.2.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |     \- xmlpull:xmlpull:jar:1.1.3.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- org.kie.kogito:kogito-services:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- org.kie.kogito:drools-core-static:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- org.kie.kogito:drools-compiler:jar:1.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- org.drools:drools-compiler:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.kie:kie-memory-compiler:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.drools:drools-ecj:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- org.kie.soup:kie-soup-maven-support:jar:7.67.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     \- org.antlr:antlr-runtime:jar:3.5.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  \- org.jboss:jandex:jar:2.4.3.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  +- com.google.guava:guava:jar:31.1-jre:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- com.google.guava:failureaccess:jar:1.0.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- org.checkerframework:checker-qual:jar:3.25.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- com.google.errorprone:error_prone_annotations:jar:2.15.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  \- com.google.j2objc:j2objc-annotations:jar:1.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  +- com.neovisionaries:nv-i18n:jar:1.27:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  +- io.quarkus:quarkus-resteasy-jackson:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- io.quarkus:quarkus-resteasy:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- io.quarkus:quarkus-resteasy-server-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- io.quarkus:quarkus-resteasy-common:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     |  \- org.jboss.resteasy:resteasy-core:jar:4.7.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     |     +- org.jboss.spec.javax.ws.rs:jboss-jaxrs-api_2.1_spec:jar:2.0.1.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     |     +- org.jboss.resteasy:resteasy-core-spi:jar:4.7.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     |     \- com.ibm.async:asyncutil:jar:0.1.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     \- jakarta.validation:jakarta.validation-api:jar:2.0.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- io.quarkus:quarkus-jackson:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- com.fasterxml.jackson.module:jackson-module-parameter-names:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- org.jboss.resteasy:resteasy-jackson2-provider:jar:4.7.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  +- com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  |  \- com.fasterxml.jackson.module:jackson-module-jaxb-annotations:jar:2.13.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- com.github.java-json-tools:json-patch:jar:1.13:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     +- com.github.java-json-tools:msg-simple:jar:1.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     |  \- com.github.java-json-tools:btf:jar:1.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     \- com.github.java-json-tools:jackson-coreutils:jar:2.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  \- org.jboss.spec.javax.xml.bind:jboss-jaxb-api_2.3_spec:jar:2.0.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  +- com.graphhopper:graphhopper-reader-osm:jar:0.13.0-pre13:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- com.graphhopper:graphhopper-core:jar:0.13.0-pre13:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.graphhopper:graphhopper-api:jar:0.13.0-pre13:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  +- com.carrotsearch:hppc:jar:0.8.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- org.apache.xmlgraphics:xmlgraphics-commons:jar:2.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |     \- commons-io:commons-io:jar:2.11.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  +- org.openstreetmap.osmosis:osmosis-osm-binary:jar:0.47:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  |  \- com.google.protobuf:protobuf-java:jar:3.19.6:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  |  \- org.slf4j:slf4j-api:jar:1.7.36:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |  \- jakarta.xml.bind:jakarta.xml.bind-api:jar:2.3.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    |     \- jakarta.activation:jakarta.activation-api:jar:1.2.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    +- org.optaweb.vehiclerouting:optaweb-vehicle-routing-frontend:war:8.13.1-SNAPSHOT:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]    \- io.quarkus:quarkus-undertow:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus:quarkus-arc:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- io.quarkus.arc:arc:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus:quarkus-vertx-http:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-security-runtime-spi:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-credentials:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-mutiny:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.quarkus:quarkus-smallrye-context-propagation:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  \- io.smallrye:smallrye-context-propagation:jar:1.2.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |     +- io.smallrye:smallrye-context-propagation-api:jar:1.2.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |     \- io.smallrye:smallrye-context-propagation-storage:jar:1.2.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.smallrye.reactive:mutiny-smallrye-context-propagation:jar:1.7.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.smallrye.common:smallrye-common-vertx-context:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.smallrye.common:smallrye-common-constraint:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-vertx-http-dev-console-runtime-spi:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-vertx:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.quarkus:quarkus-netty:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  \- com.aayushatharva.brotli4j:brotli4j:jar:1.8.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |     \- com.aayushatharva.brotli4j:native-linux-x86_64:jar:1.8.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-codec-haproxy:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.smallrye.common:smallrye-common-annotation:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.quarkus:quarkus-vertx-latebound-mdc-provider:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.smallrye.reactive:smallrye-mutiny-vertx-core:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  +- io.smallrye.reactive:smallrye-mutiny-vertx-runtime:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  \- io.smallrye.reactive:vertx-mutiny-generator:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |     \- io.vertx:vertx-codegen:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.smallrye:smallrye-fault-tolerance-vertx:jar:5.5.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.smallrye.reactive:smallrye-mutiny-vertx-web:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.smallrye.reactive:smallrye-mutiny-vertx-web-common:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.smallrye.reactive:smallrye-mutiny-vertx-auth-common:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.smallrye.reactive:smallrye-mutiny-vertx-bridge-common:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.smallrye.reactive:smallrye-mutiny-vertx-uri-template:jar:2.27.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     \- io.vertx:vertx-uri-template:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.vertx:vertx-web:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.vertx:vertx-web-common:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.vertx:vertx-auth-common:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.vertx:vertx-bridge-common:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- io.github.crac:org-crac:jar:0.1.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus.security:quarkus-security:jar:1.1.4.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- org.jboss.logging:jboss-logging:jar:3.5.0.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- io.smallrye.reactive:mutiny:jar:1.7.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |     \- org.reactivestreams:reactive-streams:jar:1.0.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- jakarta.enterprise:jakarta.enterprise.cdi-api:jar:2.0.2:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- jakarta.el:jakarta.el-api:jar:3.0.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- jakarta.interceptor:jakarta.interceptor-api:jar:1.2.5:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- jakarta.inject:jakarta.inject-api:jar:1.0:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus.http:quarkus-http-servlet:jar:4.1.9:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- jakarta.servlet:jakarta.servlet-api:jar:4.0.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- jakarta.annotation:jakarta.annotation-api:jar:1.3.5:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus.http:quarkus-http-vertx-backend:jar:4.1.9:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.vertx:vertx-core:jar:4.3.4:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-common:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-buffer:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-transport:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-handler:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  \- io.netty:netty-transport-native-unix-common:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-handler-proxy:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  |  \- io.netty:netty-codec-socks:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-codec-http2:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  +- io.netty:netty-resolver:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.netty:netty-resolver-dns:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     \- io.netty:netty-codec-dns:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- io.quarkus.http:quarkus-http-http-core:jar:4.1.9:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus.http:quarkus-http-core:jar:4.1.9:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  \- io.netty:netty-codec-http:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |     \- io.netty:netty-codec:jar:4.1.86.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       +- io.quarkus:quarkus-core:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-ide-launcher:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.quarkus:quarkus-development-mode-spi:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- io.smallrye.config:smallrye-config:jar:2.12.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |  \- io.smallrye.config:smallrye-config-core:jar:2.12.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     +- org.eclipse.microprofile.config:microprofile-config-api:jar:2.0.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     +- io.smallrye.common:smallrye-common-expression:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     |  \- io.smallrye.common:smallrye-common-function:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     +- io.smallrye.common:smallrye-common-classloader:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  |     \- io.smallrye.config:smallrye-config-common:jar:2.12.3:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- org.jboss.logmanager:jboss-logmanager-embedded:jar:1.0.10:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- org.jboss.logging:jboss-logging-annotations:jar:2.2.1.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- org.jboss.threads:jboss-threads:jar:3.4.3.Final:compile<br/>[2023-04-28T07:49:26.209Z] [INFO]       |  +- org.jboss.slf4j:slf4j-jboss-logmanager:jar:1.2.0.Final:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       |  +- org.graalvm.sdk:graal-sdk:jar:22.3.0:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       |  +- org.wildfly.common:wildfly-common:jar:1.5.4.Final-format-001:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       |  +- io.quarkus:quarkus-bootstrap-runner:jar:2.13.7.Final:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       |  |  \- io.smallrye.common:smallrye-common-io:jar:1.13.1:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       |  \- io.quarkus:quarkus-fs-util:jar:0.0.9:compile<br/>[2023-04-28T07:49:26.210Z] [INFO]       \- org.eclipse.microprofile.context-propagation:microprofile-context-propagation-api:jar:1.2:compile<br/>[2023-04-28T07:49:26.210Z] [INFO] ------------------------------------------------------------------------<br/>[2023-04-28T07:49:26.210Z] [INFO] Reactor Summary for OptaWeb Vehicle Routing 8.13.1-SNAPSHOT:<br/>[2023-04-28T07:49:26.210Z] [INFO] <br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing ............................ SUCCESS [  2.520 s]<br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing Backend .................... SUCCESS [01:01 min]<br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing Frontend ................... SUCCESS [02:49 min]<br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing Standalone ................. SUCCESS [  5.198 s]<br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing Documentation .............. SUCCESS [ 16.272 s]<br/>[2023-04-28T07:49:26.210Z] [INFO] OptaWeb Vehicle Routing Distribution ............... SUCCESS [  6.494 s]<br/>[2023-04-28T07:49:26.210Z] [INFO] ------------------------------------------------------------------------<br/>[2023-04-28T07:49:26.210Z] [INFO] BUILD SUCCESS<br/>[2023-04-28T07:49:26.210Z] [INFO] ------------------------------------------------------------------------<br/>[2023-04-28T07:49:26.210Z] [INFO] Total time:  04:22 min<br/>[2023-04-28T07:49:26.210Z] [INFO] Finished at: 2023-04-28T03:49:25-04:00<br/>[2023-04-28T07:49:26.210Z] [INFO] ------------------------------------------------------------------------<br/>[2023-04-28T07:49:26.210Z] [command]/usr/bin/docker system prune -f<br/>[2023-04-28T07:49:26.210Z] Total reclaimed space: 0B<br/>[2023-04-28T07:49:26.210Z] [INFO] Execution summary for kiegroup/optaweb-vehicle-routing<br/>[2023-04-28T07:49:26.210Z] [INFO] [BEFORE] No commands were found for kiegroup/optaweb-vehicle-routing<br/>[2023-04-28T07:49:26.210Z] # [COMMANDS] [kiegroup/optaweb-vehicle-routing] mvn clean install -f optaweb-vehicle-routing/pom.xml -Dproductized -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3  -Prun-code-coverage  -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B -s /home/jenkins/workspace/KIE/kogito/1.13.x/pullrequest/optaweb-vehicle-routing.tests.optaweb-vehicle-routing@tmp/config14141958390469393737tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true<br/>[2023-04-28T07:49:26.210Z] [INFO] OK [Executed in 264649.496527 ms]<br/>[2023-04-28T07:49:26.210Z]  <br/>[2023-04-28T07:49:26.210Z] # [AFTER] [kiegroup/optaweb-vehicle-routing] docker system prune -f<br/>[2023-04-28T07:49:26.210Z] [INFO] OK [Executed in 24.85346 ms]<br/>[2023-04-28T07:49:26.210Z]  <br/>[2023-04-28T07:49:26.210Z]  <br/>[2023-04-28T07:49:26.210Z] # Uploading artifacts<br/>[2023-04-28T07:49:26.210Z] [INFO] Will not upload any artifacts in CLI environment<br/>[2023-04-28T07:49:26.210Z]  <br/>[Pipeline] }<br/>[Pipeline] // withCredentials<br/>[Pipeline] }<br/>[2023-04-28T07:49:26.283Z] Deleting 1 temporary files<br/>[Pipeline] // configFileProvider<br/>[Pipeline] }<br/>[Pipeline] // script<br/>Post stage<br/>[Pipeline] junit<br/>[2023-04-28T07:49:26.376Z] Recording test results<br/>[2023-04-28T07:49:28.198Z] [Checks API] No suitable checks publisher found.<br/>[Pipeline] archiveArtifacts<br/>[2023-04-28T07:49:28.218Z] Archiving artifacts<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2023-04-28T07:49:30.367Z] + rm -rf console.log<br/>[Pipeline] sh<br/>
</details>

--- a/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/routing/GraphHopperRouter.java
+++ b/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/routing/GraphHopperRouter.java
@@ -33,6 +33,7 @@ import org.optaweb.vehiclerouting.service.route.Router;
 
 import com.graphhopper.GHRequest;
 import com.graphhopper.GHResponse;
+import com.graphhopper.config.Profile;
 import com.graphhopper.reader.osm.GraphHopperOSM;
 import com.graphhopper.util.PointList;
 import com.graphhopper.util.shapes.BBox;
@@ -46,11 +47,14 @@ import io.quarkus.arc.properties.IfBuildProperty;
 @IfBuildProperty(name = "app.routing.engine", stringValue = "GRAPHHOPPER", enableIfMissing = true)
 class GraphHopperRouter implements Router, DistanceCalculator, Region {
 
+    private static final String CAR_PROFILE = "car";
+
     private final GraphHopperOSM graphHopper;
 
     @Inject
     GraphHopperRouter(GraphHopperOSM graphHopper) {
         this.graphHopper = graphHopper;
+        this.graphHopper.setProfiles(new Profile(CAR_PROFILE));
     }
 
     @Override
@@ -60,6 +64,7 @@ class GraphHopperRouter implements Router, DistanceCalculator, Region {
                 from.longitude().doubleValue(),
                 to.latitude().doubleValue(),
                 to.longitude().doubleValue());
+        ghRequest.setProfile(CAR_PROFILE);
         PointList points = graphHopper.route(ghRequest).getBest().getPoints();
         return StreamSupport.stream(points.spliterator(), false)
                 .map(ghPoint3D -> Coordinates.of(ghPoint3D.lat, ghPoint3D.lon))
@@ -73,6 +78,7 @@ class GraphHopperRouter implements Router, DistanceCalculator, Region {
                 from.longitude().doubleValue(),
                 to.latitude().doubleValue(),
                 to.longitude().doubleValue());
+        ghRequest.setProfile(CAR_PROFILE);
         GHResponse ghResponse = graphHopper.route(ghRequest);
         // TODO return wrapper that can hold both the result and error explanation instead of throwing exception
         if (ghResponse.hasErrors()) {

--- a/optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/routing/GraphHopperRouterTest.java
+++ b/optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/routing/GraphHopperRouterTest.java
@@ -33,7 +33,7 @@ import org.optaweb.vehiclerouting.service.region.BoundingBox;
 
 import com.graphhopper.GHRequest;
 import com.graphhopper.GHResponse;
-import com.graphhopper.PathWrapper;
+import com.graphhopper.ResponsePath;
 import com.graphhopper.reader.osm.GraphHopperOSM;
 import com.graphhopper.storage.GraphHopperStorage;
 import com.graphhopper.util.PointList;
@@ -50,7 +50,7 @@ class GraphHopperRouterTest {
     @Mock
     private GHResponse ghResponse;
     @Mock
-    private PathWrapper pathWrapper;
+    private ResponsePath pathWrapper;
     @Mock
     private GraphHopperStorage graphHopperStorage;
 

--- a/pom.xml
+++ b/pom.xml
@@ -43,7 +43,7 @@
   </modules>
 
   <properties>
-    <version.com.graphhopper>0.13.0-pre13</version.com.graphhopper>
+    <version.com.graphhopper>1.0</version.com.graphhopper>
     <version.com.neovisionaries>1.27</version.com.neovisionaries>
     <version.frontend-maven-plugin>1.11.3</version.frontend-maven-plugin>
     <version.node>v12.16.2</version.node>

